podcasts/me-myself-ai.yaml

```diff
@@ -0,0 +1,5 @@
+---
+rss_url: https://memyselfandai.libsyn.com/rss
+offered_by: MIT Sloan Management Review
```

Comment:
Let's remove `offered_by` for this podcast. (it's optional) 

Using https://github.com/mitodl/open-podcast-data/blob/master/podcasts/sloan-counterpoints.yaml as a model, I think we set the bar high for adding an additional `offered_by`.